mysql-plugin/src/main/java/io/cdap/plugin/mysql/MysqlSink.java

```diff
@@ -31,7 +30,7 @@
  * Sink support for a MySQL database.
  */
 @Plugin(type = BatchSink.PLUGIN_TYPE)
-@Name(MysqlConstants.PLUGIN_NAME)
+@Name(MysqlUtil.PLUGIN_NAME)
```

Comment:
why did you have to move all the property names to utility class? Lets keep them in config class. I am talking only about names for configuration properties.